### ISSUE TITLE
Issues running the manager

### ISSUE THREAD

**Issue by plhrja** [damies13/rfswarm#96]
**Describe the bug**
Manager process crashes when attempted to run.

**To Reproduce**
Trying to run the manager crashes:
```
$ rfswarm 
Robot Framework Swarm: Manager 
	Version 0.8.1 
	Configuration File:  /home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/site-packages/rfswarm_manager/RFSwarmManager.ini
                                                                                                                                                                                                                                                                                         Traceback (most recent call last):
  File "/home/plhrja/anaconda3/envs/tp-swarm/bin/rfswarm", line 5, in <module>
    from rfswarm_manager.rfswarm import RFSwarm
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/site-packages/rfswarm_manager/rfswarm.py", line 6448, in <module>
    core = RFSwarmCore()
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/site-packages/rfswarm_manager/rfswarm.py", line 2075, in __init__
    self.BuildCore()
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/site-packages/rfswarm_manager/rfswarm.py", line 2095, in BuildCore
    self.BuildCorePlan()
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/site-packages/rfswarm_manager/rfswarm.py", line 2331, in BuildCorePlan
    base.addScriptRow()
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/site-packages/rfswarm_manager/rfswarm.py", line 1567, in addScriptRow
    base.gui.addScriptRow()
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/site-packages/rfswarm_manager/rfswarm.py", line 4938, in addScriptRow
    base.scriptlist[row]["TestVar"] = tk.StringVar(base.scriptlist[row]["Test"], name="row{}".format(row))
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/tkinter/__init__.py", line 540, in __init__
    Variable.__init__(self, master, value, name)
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/tkinter/__init__.py", line 372, in __init__
    self._root = master._root()
AttributeError: 'str' object has no attribute '_root'
Exception in thread Thread-3 (pln_update_graph):
Traceback (most recent call last):
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/threading.py", line 1009, in _bootstrap_inner
    self.run()
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/threading.py", line 946, in run
    self._target(*self._args, **self._kwargs)
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/site-packages/rfswarm_manager/rfswarm.py", line 4871, in pln_update_graph
    self.canvas.draw()
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/site-packages/matplotlib/backends/backend_tkagg.py", line 10, in draw
    self.blit()
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/site-packages/matplotlib/backends/backend_tkagg.py", line 13, in blit
    _backend_tk.blit(self._tkphoto, self.renderer.buffer_rgba(),
  File "/home/plhrja/anaconda3/envs/tp-swarm/lib/python3.10/site-packages/matplotlib/backends/_backend_tk.py", line 113, in blit
    photoimage.tk.call(_blit_tcl_name, argsid)
RuntimeError: main thread is not in main loop
```

Packages:
```
appdirs                            1.4.4
beautifulsoup4                     4.10.0
certifi                            2021.5.30
charset-normalizer                 2.0.6
configparser                       5.0.2
cycler                             0.10.0
docopt                             0.6.2
furl                               2.1.3
httpserver                         1.1.0
idna                               3.2
kiwisolver                         1.3.2
lxml                               4.6.3
matplotlib                         3.4.3
numpy                              1.21.3
orderedmultidict                   1.0.1
Pillow                             8.4.0
pip                                21.2.4
psutil                             5.8.0
pyparsing                          2.4.7
python-dateutil                    2.8.2
requests                           2.26.0
rfswarm-agent                      0.8.1
rfswarm-manager                    0.8.1
robotframework                     4.1.1
robotframework-pythonlibcore       3.0.0
robotframework-seleniumlibrary     5.1.3
robotframework-seleniumtestability 1.1.0
selenium                           3.141.0
setuptools                         52.0.0.post20211006
six                                1.16.0
soupsieve                          2.2.1
tqdm                               4.62.3
urllib3                            1.26.7
webdrivermanager                   0.10.0
wrapt                              1.13.1
```

Conda environment:
```
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             4.5                       1_gnu  
bzip2                     1.0.8                h7b6447c_0  
ca-certificates           2021.7.5             h06a4308_1  
certifi                   2021.5.30       py310h06a4308_0  
ld_impl_linux-64          2.35.1               h7274673_9  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 9.3.0               h5101ec6_17  
libgomp                   9.3.0               h5101ec6_17  
libstdcxx-ng              9.3.0               hd4cf53a_17  
libuuid                   1.0.3                h7f8727e_2  
ncurses                   6.2                  he6710b0_1  
openssl                   1.1.1l               h7f8727e_0  
pip                       21.2.4          py310h06a4308_0  
python                    3.10.0               h12debd9_1  
readline                  8.1                  h27cfd23_0  
setuptools                52.0.0          py310h06a4308_1  
sqlite                    3.36.0               hc218d9a_0  
tk                        8.6.11               h1ccaba5_0  
tzdata                    2021a                h5d7bf9c_0  
wheel                     0.37.0             pyhd3eb1b0_1  
xz                        5.2.5                h7b6447c_0  
zlib                      1.2.11               h7b6447c_3  
```

OS: `Ubuntu 20.04.3 LTS`

Any help on this?

**Comment by plhrja** [comment]
Downgrading to python v.3.9.7 actually seems have solved this for me. Still worth exploring what (dependency) breaks in 3.10.0?

**Comment by damies13** [comment]
Checked on Python 3.10 on Windows 10 and got same error, so this is not an OS specific Issue

**Comment by damies13** [comment]
It seems tk.StringVar has changed the way it handles optional arguments, so I used the named method and it works now in Python 3.10 without breaking other python versions

**Comment by damies13** [comment]
Fix has been merged into v0.8.2 branch and will available with that release or by pulling this branch if you need the fix urgently.
